core/arch/riscv/include/riscv.h

```diff
@@ -115,6 +115,14 @@ static inline __noprof void mb(void)
 	asm volatile ("fence" : : : "memory");
 }
 
+static inline __noprof unsigned long read_gp(void)
+{
+	unsigned long gp;
```

Comment:
= 0

note: init value is missing at line 128, but that's unrelated to this P-R.

Reply:
> note: init value is missing at line 128, but that's unrelated to this P-R.

Will fix in another PR

Reply:
@etienne-lms 
Please see https://github.com/OP-TEE/optee_os/pull/6576